src/main/java/th/in/gamer/beans/User.java

```diff
@@ -0,0 +1,35 @@
+package th.in.gamer.beans;
```

Comment:
don't use `beans` package. it's too generic.
If this bean is a response from API. please put it into `web.dto` package
and also change a class name to `UserDTO`